scripts/cleanup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
generally suggest either `#!/usr/bin/env bash` or `#!/bin/sh` as can't really guarantee `bash` is in `/bin` i think.